examples/qml_features/rust/src/serialisation.rs

```diff
@@ -36,8 +36,8 @@ mod ffi {
     pub struct Serialisation {
         #[qproperty]
         pub number: i32,
-        #[qproperty(cxx_type = "QString")]
-        pub string: UniquePtr<QString>,
+        #[qproperty]
+        pub string: QString,
     }
```

Comment:
Stretch goal (after this PR): could we get rid of DataSerde by implementing Serialize/Deserialize on QString? :thinking:

Reply:
Right in theory we can now impl traits directly for an T easier now :-)